SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -152,6 +163,8 @@ void UGlobalStateManager::ReceiveShutdownMultiProcessRequest()
 		
 		// Since the server works are shutting down, set reset the accepting_players flag to false to prevent race conditions  where the client connects quicker than the server. 
 		SetAcceptingPlayers(false);
+		SessionId = 0;
```

Comment:
One thing I just thought, we have `SessionId = 0` in both the `GlobalStateManager` and the `SpatialNetDriver`. Perhaps we should name them distinctively to prevent future confusion.

Reply:
Like URLSessionId and SpatialSessionId? NetDriver and GSM respectively...